### PR TITLE
Disable EMIT tests 2024R1

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**

### CODE REVIEW

**Review by MaxJPRey — DISMISSED** [review]
LGTM.

**Review by SMoraisAnsys — APPROVED** [review]
Do we know what extended this workflow so much ? It ran in self than 30 minutes in the last working run (https://github.com/ansys/pyaedt/actions/runs/8138121240/job/22238148636)

**Review by Samuelopez-ansys — COMMENTED** [review]
Block PR

File: pyaedt/desktop.py [hunk 1]
```diff
@@ -175,13 +175,22 @@ def _check_grpc_port(port, machine_name=""):
...
+        rand_port = random.randint(50051, 51000)
+        try:
+            sock.bind(("", rand_port))
+            sock.close()
+            return rand_port
+        except OSError:
+            tries += 1
+            if tries == 1000:
+                break
+    raise IOError("No free ports")
```
Comment (PipKat) [inline pyaedt/desktop.py:193]:
```suggestion
    raise IOError("No ports are free.")
```
String should be a complete sentence, including punctuation.

File: _unittest_solvers/test_26_emit.py [hunk 2]
```diff
@@ -18,6 +18,7 @@
 from pyaedt.modeler.circuits.PrimitivesEmit import EmitComponents
 
 test_subfolder = "T26"
+test_review_flag = True
```
Comment (MaxJPRey) [inline _unittest_solvers/test_26_emit.py:21]: I would suggest to rename the constants using upper case naming convention.
```suggestion
TEST_SUBFOLDER = "T26"
TEST_REVIEW_FLAG = True
```

**Review by maxcapodi78 — APPROVED** [review]

**Pull request merged**